src/main/groovy/org/labkey/gradle/task/DeployEmbeddedDistribution.groovy

```diff
@@ -19,21 +18,19 @@ class DeployEmbeddedDistribution extends DeployAppBase {
     void action()
     {
         deployExecutableJar()
-        deployNlpEngine(deployBinDir)
         deployPlatformBinaries(deployBinDir)
     }
 
     private void deployExecutableJar() {
-        File distributionFile = DistributionExtension.getDistributionFile(project, true)
-        boolean isTar = FilenameUtils.getExtension(distributionFile.getName()).equals("gz")
+        File distributionFile = DistributionExtension.getDistributionFile(project)
```

Comment:
This task could probably be renamed to simply `DeployDistribution`. Or it could be left as is. Either seems fine.

Reply:
Done

FYI: I've removed a few other unnecessary "embedded" references, e.g., `getEmbeddedTomcatJarPath()` -> `getLabKeyServerJarPath()`, `getEmbeddedTarArchivePath()` -> `getTarArchivePath()`